### DISCUSSION TITLE
v2.2.1

### DISCUSSION THREAD

**Discussion by edmondas** [poweradmin/poweradmin#410] [General]
Main changes:
- Support for Internationalized Domain Names (IDN) has been added.
- The schema for MySQL (MariaDB) has been [updated](https://github.com/poweradmin/poweradmin/blob/master/sql/poweradmin-mysql-update-to-2.2.1.sql).
- LDAP and SQL authentication checks have been added.

**Full Changelog**: https://github.com/poweradmin/poweradmin/compare/v2.2.0...v2.2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/poweradmin/poweradmin/releases/tag/v2.2.1'>v2.2.1</a>.</em>

**Comment by devzorg** [comment]
The project is live? looks like the last version wast postponed on unknown time.
Would like to contribute and sponsorship, but why DigitalOcean doesn't contribute it ?

**Reply by edmondas** [reply]
The project is live. I have done a lot of refactorings in order to be able to implement new features, but the current code base is hard to maintain. In addition, I examined JS frameworks to introduce new technologies. We'll see how it goes. I'm now trying to help users migrate to the new 2.2.1 version.

It's hard for me to answer why DigitalOcean doesn't contribute, as I am not sure if they are using this project. You can support the project by providing feedback on how migration went from 2.1.7 to 2.2.1 or by documenting the required steps, reporting issues, etc.

I'm also curious about what feature is most important to you right now?

**Reply by devzorg** [reply]
Iam using powerdns-admin based on Flask
But I am a php developer and would like to use something maintanable byself.
Looked into the code it has a wrong architecture, best way to use some popular framework with template engine for ui and port backend methods. It's my POV

**Reply by edmondas** [reply]
@devzorg Thanks for the opinion. Architecture is not wrong but a bit outdated :) I'm slowly moving code to better shape in order to achieve what you say. Btw, some pages already use templating.
